### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by E3V3A**
Updated README to warn about long installation time as discussed in #1124.

> Please send your pull requests the develop branch.
> Don't forget to add the change to CHANGELOG.md.

**Note**: Sometimes the development moves very fast. It is highly
recommended that you update your branch of `develop` before creating a
pull request to send us your changes. This makes everyone's lives
easier (including yours) and helps us out on the development team.
Thanks!

* Does the pull request solve a **related** issue?
* If so, can you reference the issue?
* What does the pull request accomplish? Use a list if needed.
* If it includes major visual changes please add screenshots.

### CODE REVIEW

**Comment by E3V3A** [comment]
IDK what's the probelm:

```
Running "markdownlint:all" (markdownlint) task
Warning: 
README.md: 53: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md: 55: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
 Use --force to continue.
Aborted due to warnings.
```

File: README.md [hunk 1]
```diff
@@ -28,23 +28,32 @@ MagicMirror² focuses on a modular plugin system and uses [Electron](http://elec
...
+
+**The installation step for `npm install` will take a very long time**, often with little or no terminal response!
+
+For the RPi3 this is **~10** minutes and for the Rpi2 **~25** minutes. Do not interrupt or you risk :broken_heart:.
+
+Also note that: 
+
+* `npm start` does **not** work via SSH. But you can use `DISPLAY=:0 nohup npm start &` instead. This starts the mirror on the remote display.
+
+* If you want to debug on Raspberry Pi you can use `npm start dev` which will start the MagicMirror app with Dev Tools enabled.
```
Comment (roramirez) [inline README.md]: Same here.

**Comment by E3V3A** [comment]
Actually I the markup is correct but the travis CI apparently doesn't speak GitHub markup. So this is a BS error that I don't have any control over. 

@MichMich  What do you think, force merge?

**Comment by MichMich** [comment]
Then it will keep failing for every future Travis Build, so that's not an option.

**Comment by ConnorChristie** [comment]
@E3V3A Why would you force a merge with a broken build which is easily fixable?

**Comment by E3V3A** [comment]
Fixed!

Apparently, for bullet points, `*` is not accepted by lint, but `-` is.

**Pull request merged**